packages/muelu/adapters/CMakeLists.txt

```diff
@@ -20,6 +20,12 @@ ENDIF()
 #
 IF (${PACKAGE_NAME}_ENABLE_Belos)
 
+  ASSERT_DEFINED(Belos_ENABLE_Xpetra)
+
+  IF (NOT Belos_ENABLE_Xpetra)
+    MESSAGE(FATAL_ERROR "MueLu requires the Belos Xpetra adapters in order to work with Belos. Please set \"Belos_ENABLE_Xpetra:BOOL=ON\"")
```

Comment:
@trilinos/muelu I don't like this -- see e.g., #6932.  This should either be on by default, or you shouldn't need it.  Now MueLu is forcing all of Belos to build yet again.  @vbrunini @rrdrake

Reply:
@mhoemmen Ok, I'll take another stab at fixing this.

Reply:
@cgcgcg Sorry to be a pest -- I just think it would be better to avoid the Xpetra instantiation of Belos if possible.  If you can't help it -- e.g., if you need Xpetra's special blocked operators to work -- then that's OK.

Reply:
@mhoemmen Have a look at PR #6952. (I put the automerge on it without thinking, so it already got merged.)

Reply:
@cgcgcg Thanks for letting me know!  It looks like PR #6952 just deleted the offending implementation -- is that right?  I'm totally OK with that btw : - ) .